polar-core/src/debugger.rs

```diff
@@ -21,17 +21,6 @@ impl PolarVirtualMachine {
         format!("QUERY: {}, BINDINGS: {{{}}}", query_str, bindings_str)
     }
 
-    /// Drive debugger.
-    pub fn debug_command(&mut self, command: &str) -> PolarResult<()> {
```

Comment:
Without this change, `not debug()` would not be executed on the correct Runnable (that is, it would be executed on the parent VM instead of on the Inverter's VM, which has relevant state).